components/console-backend-service/internal/domain/k8s/limitrange_service.go

```diff
@@ -33,3 +43,55 @@ func (svc *limitRangeService) List(ns string) ([]*v1.LimitRange, error) {
 
 	return result, nil
 }
+
+func (svc *limitRangeService) Create(namespace string, name string, limit gqlschema.LimitRangeInput) (*v1.LimitRange, error) {
+	var errs apierror.ErrorFieldAggregate
+
+	parsedMaxMemory, errMaxMemory := resource.ParseQuantity(*limit.Max.Memory)
```

Comment:
Maybe extract logic for errors to separate function? This is only a proposition.